logos/src/logos/main.py

```
@@ -2559,6 +2587,14 @@ async def _execute_resource_mode(
     # Extract policy
     policy = _extract_policy(headers, logos_key, body)
 
+    priority_cap: Optional[str] = None
+    if logos_key:
+        try:
+            with DBManager() as db:
+                priority_cap = db.get_process_peer_priority(logos_key)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)